user/trusty-ci-environment.md

```diff
@@ -0,0 +1,278 @@
+- sbt 0.13.9
+
+#### PHP
+
+We do not currently have any PHP versions pre-installed in the **mega**
+image. This is at the top of our list for our next update.
+
+
+But we do have [phpenv](https://github.com/phpenv/phpenv) installed and
+it's able to install the following PHP versions.
+
+- PHP 5.3
+- PHP 5.4
+- PHP 5.5
+- PHP 5.6
+- PHP 7.0.0 RCs
+
+_Note: We're unable to build **PHP 5.2** on Trusty so far, so trying to use
+it will result in a build failure when phpenv fails to compile it_
+
+#### Other runtimes
+
+Other runtimes are available to install using `apt-get` and the official
+Ubuntu packages, any third party PPAs or custom scripts that you
+manually setup in your `.travis.yml`
+
+### Data Stores
+
+We pre-install the following services and they can be activated with the built-in [services](/user/database-setup/) support.
+
+* PostgreSQL 9.4.5
+* SQLite 3.8.2
+* Redis 3.0.4
+
+*Note: The following services that are included with our Precise images
+are **not included** in the Trusty **mega** images at this time but will
+be added in the coming months. You'll need to install them manually for
+the time being.*
+
+- MySQL
+- Riak
+- ElasticSearch
+- Apache Cassandra
+- CouchDB
+- Neo4J
+
+### Firefox
+
+Firefox 31.6.0esr is installed by default.
+
+If you need a specific version of Firefox, use the Firefox addon to install
+it during the `before_install` stage of the build.
+
+For example, to install version 17.0, add the following to your
+`.travis.yml` file:
+
+    addons:
+      firefox: "17.0"
+
+### Headless Browser Testing Tools
+
+* [xvfb](http://en.wikipedia.org/wiki/Xvfb) (X Virtual Framebuffer)
+* [PhantomJS](http://phantomjs.org/)
+
+### Environment variables
+
+There is a [list of default environment variables](/user/environment-variables#Default-Environment-Variables) available in each build environment.
+
+### apt configuration
+
+apt is configured to not require confirmation (assume -y switch by default) using both `DEBIAN_FRONTEND` env variable and apt configuration file. This means `apt-get install -qq` can be used without the -y flag.
+
+### Group membership
+
+The user executing the build (`$USER`) belongs to one primary group derived from that user.
+If your project needs extra memberships to run the build, follow these steps:
+
+1. Set up the environment. This can be done any time during the build lifecycle prior to the build script execution.
+    - Set up and export environment variables.
+    - Add `$USER` to desired secondary groups: `sudo usermod -a -G SECONDARY_GROUP_1,SECONDARY_GROUP_2 $USER`
+    You may modify the user's primary group with `-g`.
+1. Your `script` would look something like:
+
+```bash
+script: sudo -E su $USER -c 'COMMAND1; COMMAND2; COMMAND3'
+```
+This will pass the environment variables down to a `bash` process which runs as `$USER`,
+while retaining the environment variables defined
+and belonging to secondary groups given above in `usermod`.
+
+### Build system information
```

Comment:
This section matches what's in precise, i'd like to avoid changing it until we overhaul how we're describing the environments